integration/activemq-vertx-integration/src/main/java/org/apache/activemq/artemis/integration/vertx/ActiveMQVertxLogger.java

```diff
@@ -38,7 +38,7 @@
  * so an INFO message would be 191000 to 191999
  */
 @MessageLogger(projectCode = "AMQ")
-public interface ActiveMQVertxLogger extends BasicLogger {
+interface ActiveMQVertxLogger extends BasicLogger {
```

Comment:
This would result in making everything public, because anyone perhaps could us it in future.
IMO it should be the other way around. Keep it as private as possible and widen access if needed.

Just say if you want to have it public or not. Will change it.